src/plugins/integrations/AWSIntegration.ts

```diff
@@ -255,7 +260,7 @@ class AWSIntegration implements Integration {
               }
             }
           } else if (serviceName === 'kinesis') {
-            const streamName = request.params ? request.params.StreamName : AWS_SERVICE_REQUEST;
+            const streamName = koalas(request.params.StreamName, AWS_SERVICE_REQUEST);
```

Comment:
If stream name is not valid for the request, it is ok to use `AWS_SERVICE_REQUEST` as the default span name, but it should not be used for `AwsKinesisTags. STREAM_NAME` tag. So you may handle this by introducing another local variable for span name

Reply:
Done. Introduced a `spanName` variable.